code/modules/hydroponics/grown/melon.dm

```diff
@@ -58,6 +58,7 @@
 	wine_power = 70 //Water to wine, baby.
 	wine_flavor = "divinity"
 
-/obj/item/reagent_containers/food/snacks/grown/holymelon/Initialize()
-	. = ..()
-	AddComponent(/datum/component/anti_magic, TRUE, TRUE) //deliver us from evil o melon god
```

Comment:
```AddComponent(/datum/component/anti_magic, FALSE , TRUE)```
So they keep their holy. Holy only protects against Watchers as well as Juggernauts 90 attack spell. As far as I know making them still weaker to stop cult hardstuns.